databricks/koalas/testing/doctest_main.py

```diff
@@ -0,0 +1,9 @@
+
+
+if __name__ == "__main__":
+    import doctest
+    import databricks.koalas as ks
+    from databricks.koalas import frame, series, namespace
+    modules = [frame, series, namespace]
+    for m in modules:
+        doctest.testmod(m, extraglobs={"ks": ks})
```

Comment:
This is fine too, for now.

One advantage of following PySpark one (which script has its own main that runs doctests) is, we can just run that the script we're fixing, for instance, in PyCharm. In this way, it's tricky to conditionally run each.

The other way might be to switch to pytest (which pandas uses), with [`conftest.py`](https://docs.pytest.org/en/latest/doctest.html#the-doctest-namespace-fixture).